dev-db/db/db18-18.1.40.recipe

```diff
@@ -0,0 +1,132 @@
+SUMMARY="The Oracle Berkeley Database"
+DESCRIPTION="The Oracle Berkeley DB family of open source, embeddable \
+databases provides developers with fast, reliable, local persistence with zero \
+administration."
+HOMEPAGE="http://www.oracle.com/technetwork/database/berkeleydb/index.html"
+COPYRIGHT="1990-2013, Oracle
+	1990-1995, The Regents of the University of California
+	1996-1996, The President and Fellows of Harvard University
+	2000-2005, INRIA - France Telecom"
+LICENSE="SleepyCat"
+REVISION="1"
+SOURCE_URI="http://download.oracle.com/berkeley-db/db-$portVersion.tar.gz"
+CHECKSUM_SHA256="0cecb2ef0c67b166de93732769abdeba0555086d51de1090df325e18ee8da9c8"
+SOURCE_DIR="db-$portVersion"
+PATCHES="db-$portVersion.patchset"
+
+ARCHITECTURES="all"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	db18$secondaryArchSuffix = $portVersion compat >= 18.1
+	lib:libdb_18$secondaryArchSuffix = $portVersion
+	lib:libdb_18.1$secondaryArchSuffix = $portVersion
+	lib:libdb_cxx_18$secondaryArchSuffix = $portVersion
+	lib:libdb_cxx_18.1$secondaryArchSuffix = $portVersion
+#	lib:libdb_java_18$secondaryArchSuffix = $portVersion
+#	lib:libdb_java_18.1$secondaryArchSuffix = $portVersion
+#	lib:libdb_java_18.1_g$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	"
+
+if [ -z "$secondaryArchSuffix" ]; then
+	SUMMARY_tools="The TIFF tools"
```

Comment:
This summary is wrong.

Reply:
Whoops! Left over from copy/paste :) (will fix)